src/main/java/com/synopsys/integration/alert/channel/azure/boards/descriptor/AzureBoardsGlobalUIConfig.java

```diff
@@ -39,20 +37,15 @@
 public class AzureBoardsGlobalUIConfig extends UIConfig {
     public static final String LABEL_AZURE_BOARDS_URL = "Url";
     public static final String LABEL_ORGANIZATION_NAME = "Organization Name";
-    public static final String LABEL_CONSUMER_KEY = "Consumer Key";
-    public static final String LABEL_PRIVATE_KEY = "Private Key";
```

Comment:
If we're doing OAuth, shouldn't Consumer and private keys always be needed in the handshake?

Reply:
Not for OAuth 2.0.